modules/vardictjava/main.nf

```diff
@@ -41,7 +42,7 @@ process VARDICTJAVA {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        vardict-java: $VERSION
+        vardict-java: \$(realpath \$(which vardict-java) | awk '{n=split(\$0,a,"/"); split(a[n-2],b,"-"); print b[3]}')
```

Comment:
```suggestion
        vardict-java: \$( realpath \$( command -v vardict-java ) | sed 's/.*java-//;s/-.*//' )
        var2vcf_valid.pl: \$( var2vcf_valid.pl -h | sed '2!d;s/.* //' )
```
This is just a personal preference to shorten these version commands as much as possible and minimize processing.